docs/administration.md

```diff
@@ -259,6 +266,14 @@ current export such as files from deleted documents, specify `--delete`.
 Be careful when pointing paperless to a directory that already contains
 other files.
 
+Paperless will not export archive files if you use `--no-archive`, or will
+not export thumbnails if you use `--no-thumbnail`. After importing, These
```

Comment:
For both the no thumbnail and no archive options, there should be a clear warning that the sanity checker will warn about missing files and the documents view will not have thumbnails until the user intervenes manually

Reply:
fix by c0466755d8b3cc0ae5dbed6993b4392d83f210c6

Reply:
I tweaked how the note was displayed